<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRasterLabelingWidgetBase</class>
 <widget class="QWidget" name="QgsRasterLabelingWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="mLabelModeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mLabelRulesButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Configure project labeling rules</string>
       </property>
       <property name="icon">
        <iconset resource="../../../images/images.qrc">
         <normaloff>:/images/themes/default/mIconLabelingRules.svg</normaloff>:/images/themes/default/mIconLabelingRules.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mEngineSettingsButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Automated placement settings (applies to all layers)</string>
       </property>
       <property name="icon">
        <iconset resource="../../../images/images.qrc">
         <normaloff>:/images/themes/default/mIconAutoPlacementSettings.svg</normaloff>:/images/themes/default/mIconAutoPlacementSettings.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="mStackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
